pkgs/applications/audio/vcv-rack/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv, fetchFromGitHub, fetchurl, pkgconfig
+, alsaLib, curl, glew, glfw, gtk2-x11, jansson, libjack2, libXext, libXi
+, libzip, rtaudio, rtmidi, speex }:
+
+let
+  # We need newer versions for some of the dependencies. They are normally
+  # checked out as submodules of the VCV Rack git repository and built for
+  # static linking. We build them as Nix packages instead.
+  rtaudio-git = rtaudio.overrideAttrs (oldAttrs: {
+    name = "rtaudio-git-20180607";
+    src = fetchFromGitHub {
+      owner = "thestk";
+      repo = "rtaudio";
+      rev = "8f4579a2f8a66e222c0b1f5a2727feff379c93d3";
+      sha256 = "0lfx6arfdid0h4kr8flpbxjicm8vnmwcsxyv1by4p76vq9bv1jjs";
+    };
+  });
+  glfw-git = glfw.overrideAttrs (oldAttrs: {
+    name = "glfw-git-20180529";
+    src = fetchFromGitHub {
+      owner = "glfw";
+      repo = "glfw";
+      rev = "0be4f3f75aebd9d24583ee86590a38e741db0904";
+      sha256 = "0zbcjgc7ks25yi949k0wjknfl00a4dqmz45mhp00k62vlq2sj0i5";
+    };
+    buildInputs = oldAttrs.buildInputs ++ [ libXext libXi ];
+  });
+in
+with stdenv.lib; stdenv.mkDerivation rec {
+  name = "VCV-Rack-${version}";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "VCVRack";
+    repo = "Rack";
+    # Because fetchSubmodules doesn't seem to work with tags we specify the
+    # revision instead
+    rev = "c6be92e5054e0aa8c4942ee7a1f0c67975fc141e";
+    sha256 = "02cm37hxyklyacjnj1ay067cm6h975xydiwij8lw6ml9sfbk33g5";
+    fetchSubmodules = true;
```

Comment:
`fetchFromGitHub` should work with tags also, what didn't work with it?
Also, is `fetchSubmodules = true` really needed? It slows it down considerably as github's tarballs can't be used.

Reply:
Oh, I don't know what I did previously, but now it works with the tagged version.

I think there are references directly to those submodule, so there would be a bit of work involved to retrieve all dependencies via Nix and patching everything up.

Reply:
I have changed that in the latest version